onset_maproom/enactstozarr.py

```diff
@@ -10,21 +10,22 @@
 
 RR_MRG_NC_PATH = CONFIG["rr_mrg_nc_path"]
 RR_MRG_ZARR_PATH = CONFIG["rr_mrg_zarr_path"]
+RESOLUTION = CONFIG["rr_mrg_resolution"]
+CHUNKS = CONFIG["rr_mrg_chunks"]
 
 #Read daily files of daily rainfall data
 #Concatenate them against added time dim made up from filenames
 
 RR_MRG_PATH = Path(RR_MRG_NC_PATH)
 RR_MRG_FILE = list(sorted(RR_MRG_PATH.glob("*.nc")))
 
-RESOLUTION = .25
-
 
 def set_up_dims(xda):
+    datestr = Path(xda.encoding["source"]).name.split("_")[2]
```

Comment:
hopefully this will work for a good deal of ENACTS datasets (I mean 3rd of split by _)